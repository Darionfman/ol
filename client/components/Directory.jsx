import React from 'react'
import * as bs from 'react-bootstrap'
import {observer} from 'mobx-react'
import ListItem from './ListItem'
import colectionStore from '../store'

@observer
export default class Directory extends React.Component {
  constructor (props) {
    super(props)
  }

  componentDidMount(){
    colectionStore.fetchFirstPage()
  }

  fetch(link){
    if(!!link) colectionStore.fetchPage(link)
  }

  render () {
    let list = colectionStore.store
    let nextLink = colectionStore.next
    let prevLink = colectionStore.previous
    if(!list.length) return <div> Loading </div>

    let directories = list.map((business) => (<ListItem key={business.id} business={business} />))
    return (
      <bs.Col md={12}>
        <bs.Row>
          <bs.Col md={6}>
          <bs.Button onClick={() => this.fetch(prevLink)}>previous</bs.Button>
          </bs.Col>
          <bs.Col md={6}>
          <bs.Button className="right" onClick={() => this.fetch(nextLink)}>next</bs.Button>
          </bs.Col>
        </bs.Row>
        <bs.Row>
          {directories}
        </bs.Row>
      </bs.Col>
    )
  }
}