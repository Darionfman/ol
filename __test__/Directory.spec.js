import React from 'react'
import Directory from '../client/components/Directory'
import Store from '../client/store'
import renderer from 'react-test-renderer'

test('Directory renders with correct number of children', async () => {
  await Store.fetchFirstPage()
  
  const component = renderer.create(
    <Directory />
  )
  let tree = component.toJSON()
  //should have all 50 businesses as child nodes
  expect(tree.children.length).toBe(50)
  //child nodes should be contained in divs
  expect(tree.children[0].type).toBe('div')
  //they should be full width collumns
  expect(tree.children[12].props.className).toBe('col-md-12')
  //the second child of the child nodes should have a child containing text
  expect(typeof tree.children[12].children[0].children[1].children[0]).toBe('string')
  //checks to make sure it matches the recent snapshot
  expect(tree).toMatchSnapshot()
})