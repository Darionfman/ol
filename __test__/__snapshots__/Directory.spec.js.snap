exports[`test Directory renders with correct number of children 1`] = `
<div
  className="col-md-12">
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Yundt-Flatley
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.halvorson.com/">
        http://www.halvorson.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4034880719">
        4034880719
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Botsford Ltd
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://bergstrom.org/">
        http://bergstrom.org/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:2462288476">
        2462288476
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Turner, Kris and Conroy
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://schmitt.net/">
        http://schmitt.net/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1788019885">
        1788019885
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Kertzmann LLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://okeefe.info/">
        http://okeefe.info/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4442359224">
        4442359224
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Nader-Mosciski
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://schumm.com/">
        http://schumm.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7106578275">
        7106578275
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Renner, Ferry and Dibbert
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.schiller-bartell.net/">
        http://www.schiller-bartell.net/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:5491562078">
        5491562078
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Fritsch and Sons
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.schmeler.com/">
        http://www.schmeler.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:6068112791">
        6068112791
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Bins-Bosco
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.cartwright-white.com/">
        http://www.cartwright-white.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1516769808">
        1516769808
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Mueller, Davis and Pfeffer
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.schaefer.com/">
        http://www.schaefer.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:8827125278">
        8827125278
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Lockman-Ward
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://legros.org/">
        http://legros.org/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7888247882">
        7888247882
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Bashirian LLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.windler.info/">
        http://www.windler.info/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1366898018">
        1366898018
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Jacobson, Bashirian and Hansen
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://collins.biz/">
        http://collins.biz/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:5858963747">
        5858963747
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Waters, Larkin and Nienow
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://upton.com/">
        http://upton.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:6793339715">
        6793339715
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Parker LLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.kertzmann.com/">
        http://www.kertzmann.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4778921820">
        4778921820
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Pollich PLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.collier.net/">
        http://www.collier.net/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4257221865">
        4257221865
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Dare PLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.rosenbaum-gerhold.com/">
        http://www.rosenbaum-gerhold.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:2250907292">
        2250907292
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Fritsch PLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.gleichner-fay.com/">
        http://www.gleichner-fay.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4874514201">
        4874514201
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Rogahn, Armstrong and Towne
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://ortiz.net/">
        http://ortiz.net/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4634023657">
        4634023657
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Nikolaus, Batz and Hyatt
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://jacobs.com/">
        http://jacobs.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:6045870402">
        6045870402
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Smitham Group
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.botsford.com/">
        http://www.botsford.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:5962018046">
        5962018046
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Anderson LLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.weissnat.com/">
        http://www.weissnat.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:2070582456">
        2070582456
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Runolfsson PLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.goldner.com/">
        http://www.goldner.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:4197749267">
        4197749267
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Schamberger, Nolan and Mann
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://west.net/">
        http://west.net/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:5138313444">
        5138313444
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Miller, Mraz and Beatty
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://cormier.com/">
        http://cormier.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:8544861760">
        8544861760
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Harvey and Sons
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.ziemann.com/">
        http://www.ziemann.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:9150464732">
        9150464732
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Predovic, Hammes and Jenkins
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.okon.com/">
        http://www.okon.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:8450811011">
        8450811011
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Schroeder Ltd
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://farrell-hettinger.info/">
        http://farrell-hettinger.info/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:6835570001">
        6835570001
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Bayer-Beahan
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://maggio.info/">
        http://maggio.info/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1968006576">
        1968006576
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Nienow-Gerlach
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.keeling.com/">
        http://www.keeling.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7129561196">
        7129561196
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Jaskolski, Marquardt and Mueller
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://feil.com/">
        http://feil.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7356889694">
        7356889694
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Crist, Kuhlman and Stiedemann
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://rutherford-durgan.com/">
        http://rutherford-durgan.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7271068051">
        7271068051
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Kirlin, Mohr and Wisozk
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.koss.org/">
        http://www.koss.org/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:0827821757">
        0827821757
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Quigley Group
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.ebert.com/">
        http://www.ebert.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1187452299">
        1187452299
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Mraz and Sons
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.runte.com/">
        http://www.runte.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:9543464624">
        9543464624
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Mante, Dicki and Satterfield
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://aufderhar.com/">
        http://aufderhar.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:0303519618">
        0303519618
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Stroman, Halvorson and Denesik
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://cruickshank.com/">
        http://cruickshank.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:2378157628">
        2378157628
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Ankunding Group
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.stoltenberg.com/">
        http://www.stoltenberg.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:3632256643">
        3632256643
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Denesik-Bogisich
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://kunze.biz/">
        http://kunze.biz/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1134687127">
        1134687127
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Hegmann-Lowe
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://braun-nicolas.com/">
        http://braun-nicolas.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:6949336515">
        6949336515
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Stoltenberg, Muller and Dickens
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.mclaughlin.com/">
        http://www.mclaughlin.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7211056265">
        7211056265
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Heller-Lind
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.green-cassin.com/">
        http://www.green-cassin.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:0249816319">
        0249816319
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Bernier, Labadie and Kuvalis
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.upton.com/">
        http://www.upton.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:7529901027">
        7529901027
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Lesch, Rath and Larson
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://connelly-renner.com/">
        http://connelly-renner.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:5991751665">
        5991751665
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Hansen-Reynolds
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://www.schmitt-lueilwitz.org/">
        http://www.schmitt-lueilwitz.org/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:2292139990">
        2292139990
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Rempel, McGlynn and Abshire
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://mayert.com/">
        http://mayert.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:9947828030">
        9947828030
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Becker LLC
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://beer.org/">
        http://beer.org/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:5431345849">
        5431345849
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        West, Jaskolski and Stroman
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://schowalter.info/">
        http://schowalter.info/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:6164509215">
        6164509215
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Muller Ltd
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://jones-howe.com/">
        http://jones-howe.com/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:2633424635">
        2633424635
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Grimes Group
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://ledner.org/">
        http://ledner.org/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:8677489190">
        8677489190
      </a>
    </div>
  </div>
  <div
    className="col-md-12">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="col-md-2">
        Ol
      </div>
      <div
        className="col-md-3">
        Monahan Ltd
      </div>
    </a>
    <div
      className="col-md-4">
      <a
        href="http://smitham.info/">
        http://smitham.info/
      </a>
    </div>
    <div
      className="col-md-3">
      <a
        href="tel:1872318358">
        1872318358
      </a>
    </div>
  </div>
</div>
`;
